trade-finance-manager-api/src/v1/mappings/map-eligibility-criteria-content-strings.js

```diff
@@ -5,13 +5,12 @@ const mapEligibilityCriteriaContentStrings = (eligibility, dealType) => {
   const mappedCriteria = eligibility.criteria;
   const contentStrings = CONTENT_STRINGS.DEAL.ELIGIBILITY_CRITERIA[dealType];
 
-  const versionContentStrings = contentStrings[eligibility.version];
+  const versionContentStrings = contentStrings[Number(eligibility.version)];
```

Comment:
Yes it should be number as the array index are defined in as a number rather than a string.
This to be honest wont make any difference more of a safety net.